src/picterra/client.py

```diff
@@ -869,6 +873,7 @@ def upload_vector_layer(
             raster_id: The id of the raster we want to attach the vector layer to
             filename: Path to the local GeoJSOn file we want to upload
             name: Optional name to give to the vector layer
+            color: Optional color of the vector layer
```

Comment:
Some details here on what format the color is in (is this a html hex code?) would be useful to document I feel

Reply:
fixed